### PR TITLE
Refactor uefistored conftest

### PR DESCRIPTION
**Pull request by benjamreis**
New fixture `unix_uefi_vm_and_snapshot`

Signed-off-by: BenjiReis <benjamin.reis@vates.fr>

### CODE REVIEW

**Comment by benjamreis** [comment]
Tests ok on linux and Windows

File: tests/uefistored/conftest.py [hunk 1]
```diff
@@ -18,10 +18,16 @@ def pool_without_uefi_certs(host):
...
 
 @pytest.fixture(scope='module')
-def uefi_vm_and_snapshot(imported_vm):
+def uefi_vm(imported_vm):
     vm = imported_vm
     if not vm.is_uefi:
-        pytest.skip('imported_sb_vm can only be used on UEFI VMs')
+        pytest.skip('uefi_vm can only be used on UEFI VMs')
+
+    yield vm
```
Comment (stormi) [inline tests/uefistored/conftest.py:26]: We can move it to the global conftest.py now.

File: tests/uefistored/conftest.py [hunk 2]
```diff
@@ -31,19 +37,18 @@ def uefi_vm_and_snapshot(imported_vm):
...
-    vm, _ = uefi_vm_and_snapshot
-    yield vm
-
 @pytest.fixture(scope='module')
 def linux_uefi_vm(uefi_vm):
     vm = uefi_vm
     if vm.is_windows:
         pytest.skip("linux_uefi_vm can only be used on Linux VMs")
 
     yield vm
```
Comment (stormi) [inline tests/uefistored/conftest.py:50]: While we're at refactoring, this fixture is not necessary anymore. It's just a combination of `unix_vm` (which will be available in global conftest once Gaël's PR is merged) and `uefi_vm`, and we can use them instead.

**Review by gduperrey — APPROVED** [review]

File: tests/uefistored/conftest.py (hunk 2 above)
Comment (benjamreis) [reply tests/uefistored/conftest.py:42]: Should it be named `unix_...` as well?

Comment (stormi) [reply tests/uefistored/conftest.py:42]: Probably

**Review by stormi — APPROVED** [review]

**Pull request merged**